### DISCUSSION TITLE
Series with 2 different narrators

### DISCUSSION THREAD

**Discussion by SenorSmartyPants** [seanap/Plex-Audiobook-Guide#72] [General]
I have the full series of Harry Potter audiobooks, US and UK versions. Narrated by Jim Dale and Stephen Fry respectively. Audible has all these books in the same series which is of course correct. But I'm wondering about tagging them or the directory structure for them. I've thought of editing the series name to be `Harry Potter (US)` or with narrator name. 

How have others dealt with similar problems (if anyone else has)?

**Comment by MartinCa** [comment]
I have dealt with a similar issue with Graphic Audio audiobooks (dramatized). The way I dealt with it was to suffix the series and the book title with (Dramatized). The reason I did it with both the series and the book title is that I wanted to be able to see it no matter if I was looking at the book alone or in the series context.

Not sure what the "correct" way of dealing with it is, but that was the way I dealt with it.
